plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -959,11 +959,18 @@
 }
 
 #variable_product_options #message,
+#inventory_product_data .notice,
 #variable_product_options .notice {
 	display: flex;
 	margin: 10px;
 	background-color: #ffffff;
 	> p {
+		a {
+			text-decoration: none;
+			white-space: nowrap;
+		}
+	}
+	> p:not( :last-child ) {
```

Comment:
As mentioned, this explicit width causes issues but it outside the scope of this PR, so I'm good with this fix for now!